plasma_framework/contracts/src/exits/payment/routers/PaymentStandardExitRouter.sol

```diff
@@ -40,6 +40,8 @@ contract PaymentStandardExitRouter is
     PaymentChallengeStandardExit.Controller internal challengeStandardExitController;
     BondSize.Params internal startStandardExitBond;
 
+    address private maintainer;
```

Comment:
Can we consider using framework.getMaintainer() in onlyFrom() instead of creating another storage variable?

Reply:
lol, just realized we are not saving `framework` in the `Router` to storage xdd. so...shall we replace the `maintainer` with `framework` instead XDD?

Reply:
yeah